sup3r/pipeline/pipeline.py

```diff
@@ -48,80 +44,3 @@ def __init__(self, pipeline, monitor=True, verbose=False):
         if 'logging' in self._config:
             init_logger('sup3r.pipeline', **self._config.logging)
             init_logger('reV.pipeline', **self._config.logging)
-
-    @classmethod
-    def init_pass_collect(cls, out_dir, file_paths, model_path,
-                          fwp_kwargs=None, dc_kwargs=None):
-        """Generate config files for forward pass and collection
```

Comment:
Never used? Auto-making configs is often redundant because you need to organize the inputs to make the configs in the first place haha.

Reply:
Haha yeah forgot I had even added this and never used it